README.md

```diff
@@ -1 +1,76 @@
-# angular-what-is-a-filter-readme
\ No newline at end of file
+# What is a Filter?
+
+## Overview
+
+Angular offers us the powerful concept of filters. Let's take a look at what they can offer us.
```

Comment:
Add more context here - why are filters so cool? Give the elevator pitch for why this is an awesome feature and reference a real world example. Use this overview as the moment to get the reader excited about learning to use filters.